plugins/nanospell/plugin.js

```diff
@@ -144,6 +140,10 @@
 			return i;
 
 		},
+		normalizeWord: function (word) {
+			// hex 200b = 8203 = zerowidth space
+			return word.replace(/\u200B/g, '');
+		},
```

Comment:
for my own sanity, sometimes when you remove wrapping elements (causing text nodes to neighbor each other) the text nodes then contain zero width spaces.  Which makes string comparisons mysteriously fail